std/datetime/package.d

```diff
@@ -148,18 +147,31 @@ import std.typecons : Flag, Yes, No;
 @safe unittest
 {
     assert(std.datetime.Date(2010, 9, 7) + dur!"days"(5) ==
-           std.datetime.Date(2010, 9, 12));
+            std.datetime.Date(2010, 9, 12));
 
     assert(std.datetime.Date(2010, 9, 7) - std.datetime.Date(2010, 10, 3) ==
-           dur!"days"(-26));
+            dur!"days"(-26));
 }
 
 @safe unittest
 {
     import std.traits : hasUnsharedAliasing;
+
     /* https://issues.dlang.org/show_bug.cgi?id=6642 */
     static assert(!hasUnsharedAliasing!Date);
     static assert(!hasUnsharedAliasing!TimeOfDay);
     static assert(!hasUnsharedAliasing!DateTime);
     static assert(!hasUnsharedAliasing!SysTime);
 }
+
+auto now(DT)(immutable TimeZone dz = LocalTime()) @safe nothrow @nogc
```

Comment:
I don't understand why this function is needed.

Reply:
@Tynukua can you please provide a rationale for why this function is needed? We have a policy of preventing one line functions to make it into phobos. We do make exceptions for chained functions call that appear frequently, but this does not seem to be case in this scenario.